pkg/users/account-create-dialog.js

```diff
@@ -272,15 +326,19 @@ export function account_create_dialog(accounts, shells) {
                             "-e",
                             user_name
                         ], { superuser: "require", err: "message" });
+                })
+                .then(() => {
+                    if (force_home)
+                        return cockpit.spawn(["chown", "-hR", `${user_name}:${user_name}`, home_dir], { superuser: "require", err: "message" });
```

Comment:
Are you sure that the newly created user is always in a group of the same name? I thought you could configure useradd etc. to put users into e.g. an "users" group by default.

Reply:
```
If user group is not specified, the behavior of useradd will depend on the USERGROUPS_ENAB variable in /etc/login.defs. If this variable is set to yes (or -U/--user-group is specified on
           the command line), a group will be created for the user, with the same name as her loginname. If the variable is set to no (or -N/--no-user-group is specified on the command
           line), useradd will set the primary group of the new user to the value specified by the GROUP variable in /etc/default/useradd, or 1000 by default.
```
So we could try to predict which group user was added to, but since by this point user is already create, maybe safest option is to just run "id {username}" and check the `gid` of the user?